app/main/views/drafts.py

```diff
@@ -139,6 +139,8 @@ def list_draft_services():
         framework = Framework.query.filter(
             Framework.slug == framework_slug
         ).first()
```

Comment:
`.first_or_404` would've worked just fine, I think we use it quite a lot in other places